services/wikipedia-service/src/index.ts

```diff
@@ -1,34 +1,32 @@
-import initAspecto from "@aspecto/opentelemetry";
+import initAspecto from '@aspecto/opentelemetry';
 initAspecto({
   aspectoAuth:
-    process.env.ASPECTO_AUTH ?? "e97d7a26-db48-4afd-bba2-be4d453047eb",
-  local: process.env.NODE_ENV !== 'production',
+    process.env.ASPECTO_AUTH ?? 'e97d7a26-db48-4afd-bba2-be4d453047eb',
   logger: console,
```

Comment:
another thought.
Since `logger` is used only for live flows, I suggest to remove it as well from the 3 packages